k8s/pico-apiserver.yaml

```diff
@@ -73,6 +73,7 @@ spec:
             - --namespace=default
             - --ssh-username=sandbox
             - --ssh-port=22
+            - --runtime-class-name=
```

Comment:
cc @YaoZengzeng 
```suggestion
            - --runtime-class-name=""
```

Reply:
It would not work as it will be parsed as '""'